src-test/test.js

```diff
@@ -1,25 +1,27 @@
 "use strict";
 
-const fs = require("fs");
+const fs = require("fs/promises");
```

Comment:
I'm using `require("fs/promises")` here because it is what is recommended by the Node.JS documentation: see https://nodejs.org/docs/latest-v18.x/api/fs.html#promises-api

However, if we want to support Node.JS v10 and v12 (possibly not needed, since this is just test code), we can change to:
```suggestion
const fs = require("fs").promises;
```